Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Build cadence binaries
```

Comment:
Do we have to put the Dockerfiles in root directory? It would be nice if we could keep it inside docker folder.

Reply:
It's kind of a standard for Dockerfiles to be in the repository root. Also, I wanted to keep as much of the existing setup as possible, so that any kind of automated builds you might have didn't break.

It's not a requirement, but as I said, it's the standard place.

Reply:
I see, let's follow the standard then.

Reply:
Is it also standard to have Dockerfile.allinone and Dockerfile.CLI in the root? Can we only have one? Otherwise the root looks messy.